src/common/definitions.js

```diff
@@ -53,14 +53,7 @@ const rejectOnlyWhitespace = {
 
 const usaPostalCode = {
   type: 'string',
-  anyOf: [
-    {
-      pattern: '^\\d{5}$',
-    },
-    {
-      pattern: '^\\d{5}-\\d{4}$',
-    },
-  ],
+  pattern: '^\\d{5}(-\\d{4})?$',
```

Comment:
Removed `anyOf` since zipCode frontend tooling does not account for this use case of `react-JSON-schema`

Reply:
I see another "postalCode" regex that is very similar to this already the codebase. Maybe we should use the same pattern to be consistent?

`^\\d{5}(?:([-\\s]?)\\d{4})?$`